crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -1901,11 +1905,93 @@ pub(crate) fn report_invalid_exception_cause(context: &InferContext, node: &ast:
     ));
 }
 
-pub(crate) fn report_base_with_incompatible_slots(context: &InferContext, node: &ast::Expr) {
-    let Some(builder) = context.report_lint(&INCOMPATIBLE_SLOTS, node) else {
+pub(crate) fn report_class_with_multiple_solid_bases(
```

Comment:
May as well match the diagnostic code here, considering its also shorter:

```suggestion
pub(crate) fn report_instance_layout_conflict(
```